WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/EditAttributesViewController.swift

```diff
@@ -17,7 +17,7 @@ final class EditAttributesViewController: UIViewController {
     ///
     var onVariationCreation: ((Product) -> Void)?
 
-    /// Assign this closure to be notified after an attribute  is created.
+    /// Assign this closure to be notified after an attribute  is created or updated.
```

Comment:
nit: maybe this closure could be called `onAttributeUpdate`? 😄 up to you!

Reply:
Totally!